lib/chutney/TorNet.py

```diff
@@ -633,11 +640,26 @@ def _getBridgeLines(self):
         if not self._env['bridge']:
             return ""
 
-        bridgelines = "Bridge %s:%s\n" % (self._env['ip'],
-                                          self._env['orport'])
+        if self._env['pt_bridge']:
+            port = self._env['ptport']
+            transport = self._env['pt_transport']
+            extra = self._env['pt_extra']
+        else:
+            port = self._env['orport']
+            transport = ""
+            extra = ''
```

Comment:
I had to stare at these two lines for a while, before I realised they were both empty strings.

Nitpick: let's use single quotes for both.

Reply:
Fixup as f79dedc397609d

Reply:
(whoops, didn't notice that you had suggested single quotes. I went with double; I hope that's ok)